style.css

```diff
@@ -0,0 +1,13 @@
+body {
+  font-family: Montserrat;
+  font-style: normal;
+}
+
+/*.intro__title{
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 110px;
+  color: #000000;
+}*/
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6002411)